### ISSUE TITLE
Write guideliness on testimony

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/engineering-progression#207]
From S1 workshop:

A lot of the competencies' evidence requirements lean heavily on testimony. There's concern that this overly favours e.g. extroverted people who are good at talking about themselves. Particularly a concern as - in tech - line managers may not work on the same projects as their reports.

Can we provide guidelines to line managers on what constitutes valid testimony and what their responsibilities are with regard to verifying it e.g. if your report says 'I was central in delivering X', check with their product owner & tech lead that they agree with this assessment

**Comment by alicebartlett** [comment]
I think this already exists in the form of words about feedback from HR eg https://financialtimes.looop.co/#/search/feedback. We should not be deciding what good feedback looks like, HR+Learning have already done it so let's link to that.

**Comment by wheresrhys** [comment]
Yep agree. This looks like the most relevant article https://financialtimes.looop.co/#/topic/22629

**Comment by rowanmanning** [comment]
OK we've agreed we should link to relevant articles on this subject in the FAQ or the "how to use this framework" guide. Might be a good thing to pick up for someone wanting to contribute?

**Comment by rowanmanning** [comment]
We may need to link to more things that are tangentially related, based on Sofia feedback. For example a link to training resources, mentoring stuff, etc.

**Comment by mark-barnes** [comment]
feel this is more about line manager skill.  are they able to recognise good work and skills/competency development (and therefore the possibility of promotion).  regardless of extrovert/introvert nature of a direct report.    i am not sure this is with in the scope of the engineering progression framework.  propose closing this issue and pointing interested parties at looop for training on line management

**Comment by rowanmanning** [comment]
(adding some activity to stop stalebot from closing this)

**Comment by NatashaMcCann** [comment]
https://github.com/Financial-Times/engineering-progression/pull/323
